### DISCUSSION TITLE
Support for AMD please

### DISCUSSION THREAD

**Discussion by thebongmaster** [carson-katri/dream-textures#248] [Ideas]
all these AI projects are CUDA-centric and claim openness, but AMD has proven just as capable at AI generated art. and not supporting or encouraging support for both GPU types isn't truly open. all we ask is equal support and not to be left out of the CUDA walled garden. 
This AI stuff is great, but I for one am not going to buy another GPU in order to use it when the one i have is already a good GPU.
Please encourage cross architecture support.

**Comment by dunx1976** [comment]
I agree, it needs to be HIP based (maybe) rather than CUDA, but until it gets implemented at the base code, we have to wait... till then, you can use this;
https://huggingface.co/spaces/stabilityai/stable-diffusion
 Just type "XXXXX 2d texture, seamless" (brick wall 2d texture, seamless,) etc

**Comment by carson-katri** [comment]
I would gladly merge any contributions for AMD support. But not having the hardware myself, there is not much I can do to support it.

**Reply by Nacon** [reply]
SD can run on AMD via ONNX on Windows.

**Comment by PyrateGFXProductions** [comment]
I second for AMD support. This tech is super fun to play with. I am currently generating via Gen 3 I7 CPU and the time is excruciating...+20 min...
